src/app/store/index.ts

```diff
@@ -15,7 +15,6 @@ import {
 } from 'redux-persist';
 import { PersistPartial } from 'redux-persist/es/persistReducer';
 
```

Comment:
Maybe we can update `IS_DEV_ENV` to be `process.env.NODE_ENV === 'development'`? 

Or else just delete it if it's not used?

I see it's used in `remote-config.query.ts` but I'm not sure if it's working / needed

Reply:
I'm not sure why, but you should put `process.env.NODE_ENV === 'development'` directly in file where devtools library is used, otherwise webpack include it in bundle. you can try and test it, rather weird thing

Reply:
OK cool 👍